src/app/give-help/material-aid-form/material-aid-form.component.ts

```diff
@@ -24,13 +24,19 @@ export class MaterialAidFormComponent {
   phonePrefix: string = '48';
   phoneNumber: string = '';
   loading: boolean = false;
+  @ViewChild('phoneInput')
+  phoneInput!: { nativeElement: { value: any } };
+
   constructor(
     private router: Router,
     private materialAidResourceService: MaterialAidResourceService,
     private snackbarService: SnackbarService
   ) {}
 
-  onPhoneNumberChange(): void {
+  onPhoneNumberChange($event: any): void {
+    let val = $event.target.value;
+    val = val.replace(/[^0-9 ]+/g, '');
```

Comment:
To samo pytanie, co wyżej.

BTW, można by tę funkcję przenieść do jakichś utilsów, bo jest tu jakaś logika.